pkgs/development/compilers/myrddin/default.nix

```
@@ -0,0 +1,56 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, bison
+, binutils
+, binutils-unwrapped
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "myrddin";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    repo = "mc";
+    owner = "oridb";
+    rev = "r${version}";
+    sha256 = "7ImjiG/rIKGPHq3Vh/mftY7pqw/vfOxD3LJeT87HmCk=";
+  };
+
+  nativeBuildInputs = [
+    bison
+    pkg-config
+    makeWrapper
+  ];
+
+  postPatch = ''
+    sed -i 's/-Werror//g' mk/c.mk
```

```suggestion
    sed -i 's/-Werror//g' mk/c.mk
```
slightly opinionated: maybe use `substitueInPlace` instead